mirrord-layer/src/lib.rs

```diff
@@ -375,6 +392,34 @@ async fn poll_agent(mut pf: Portforwarder, mut receiver: Receiver<HookMessage>)
 
     let mut ping = false;
 
+    if config.enabled_override_env_vars {
+        let env_vars_filter = HashSet::from(EnvVarsFilter(config.override_env_vars_exclude));
+        let env_vars_select = HashSet::from(EnvVarsFilter(config.override_env_vars_include));
+
+        let env_vars_intersection = env_vars_filter.intersection(&env_vars_select);
```

Comment:
You shouldn't bother with the intersection - it doesn't make any sense for both INCLUDE and EXCLUDE to be populated at all at the same time, even if the values don't intersect. e.g. INCLUDE=a,b,c, EXCLUDE=x,y,z - if values a,b,c are included, then of course values x,y,z are excluded, there's no need to specify.

Reply:
Also, there's no need for enabled_override_env_vars to be required for them to take effect - if I specified INCLUDE or EXCLUDE then naturally I want env vars to be enabled.